source/agora/consensus/data/Block.d

```diff
@@ -334,25 +335,11 @@ public struct Block
 
         const size_t size = min(this.txs.length, this.merkle_tree.length);
```

Comment:
I am not sure why this constant is here though. The merkle tree's length should be `(tx.length * 2) - 1`, so I think this should be moved to the assert above as a check and this `min` should be removed.